pkgs/development/libraries/wlroots/default.nix

```diff
@@ -6,15 +6,15 @@
 
 let
   pname = "wlroots";
-  version = "0.5.0";
+  version = "0.6.0";
```

Comment:
I realise you didn't write this but is there any reason pname and version can't go inside the derivation and the `name = "${pname}-${version}"; line removed

Reply:
No, I don't think there is. I'll try it and see if anything breaks.

Reply:
(It's probably legacy though, because automatic pname-version is pretty new)